models/inventory-entry/src/index.ts

```diff
@@ -1,6 +1,27 @@
+import {
+  RestModelBuilder,
+  GraphqlModelBuilder,
+  CompatInventoryEntryModelBuilder,
+} from './builders';
+import * as modelPresets from './presets';
+
 export * as InventoryEntryDraft from './inventory-entry-draft';
```

Comment:
We need to export the three models with heir own names:
```suggestion
export * from './inventory-entry-draft';
```